src/bio_helper.h

```diff
@@ -28,6 +28,12 @@
         // block_device_operations as submit_bio function.
         // See https://github.com/torvalds/linux/commit/c62b37d96b6eb3ec5ae4cbe00db107bf15aebc93
         #define USE_BDOPS_SUBMIT_BIO
+
```

Comment:
mrf_type.h proposed on branch https://github.com/natalia-zelazna/dattobd/tree/addr_brutforce_test allows to cut off from this ifdef, we can consider it once this PR is closed

Reply:
I think we should make a refactoring ticket to try to get rid of ifdefs, not only this on